cuequivariance_torch/cuequivariance_torch/primitives/symmetric_tensor_product.py

```diff
@@ -335,9 +335,9 @@ def forward(
         i0 = i0.to(torch.int32)
         x0 = x0.reshape(x0.shape[0], x0.shape[1] // self.u, self.u)
         x1 = x1.reshape(x1.shape[0], x1.shape[1] // self.u, self.u)
-        logger.debug(
-            f"Calling SymmetricTensorContraction: {self.descriptors}, input shapes: {x0.shape}, {i0.shape}, {x1.shape}"
-        )
+        # logger.debug(
+        #     f"Calling SymmetricTensorContraction: {self.descriptors}, input shapes: {x0.shape}, {i0.shape}, {x1.shape}"
+        # )
```

Comment:
Shall we do the if is scripting trick here too?

Reply:
Sure!